### ISSUE TITLE
[TypeScript] A missing property in type declare file

### ISSUE THREAD

**Issue by sectsect** [typekit/webfontloader#428]
A missing property `loadAllFonts` in type declare file on [@types/webfontloader](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webfontloader/index.d.ts)

@ [Line: 55 - 58](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9d29adedf662de685356f711951ef8b9e8342865/types/webfontloader/index.d.ts#L55-L58)

```ts
export interface Monotype {
  projectId?:string;
  version?:number;
}
```

Should be added as follows:
```diff
export interface Monotype {
  projectId?:string;
  version?:number;
+ loadAllFonts?:boolean;
}
```

## FYI
The type declaration file on [@types/webfontloader](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webfontloader/index.d.ts) seems to be a file for **v1.6.4**.

**Comment by sectsect** [comment]
I submitted PR to DefinitelyTyped https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44776
It will probably be merged in the next few days.

**Comment by sectsect** [comment]
:tada: The PR has been merged and release to npm as v1.6.31. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44776

https://www.npmjs.com/package/@types/webfontloader
